mitodl/micromasters
# Hide photo upload on public profile (#1603)

#### What are the relevant tickets?

Fixes #1603 

#### What's this PR do?

This PR hides the image upload button on non-private profile pages when viewed by anyone other than the logged in user whose profile is being viewed.

#### How should this be manually tested?

Easiest way to test this is to set your profile to public and then view it anonymously. You should not see the camera icon unless you're logged in as that user.

#### Screenshots (if appropriate)

Logged in and viewing own profile:
![selection_008](https://cloud.githubusercontent.com/assets/28598/20114203/5beae1d8-a5c1-11e6-9857-ef9ac33dbd5c.png)

Anonymous or viewing someone else's profile:
![selection_009](https://cloud.githubusercontent.com/assets/28598/20114210/60c2b276-a5c1-11e6-9b64-75ed96df305c.png)
